dask-gateway/Dockerfile

```diff
@@ -2,27 +2,34 @@
 FROM debian:buster-slim as miniconda
 LABEL MAINTAINER="Jim Crist-Harif"
 
-ARG CONDA_VERSION=py38_4.8.3
-ARG CONDA_SHA256=879457af6a0bf5b34b48c12de31d4df0ee2f06a8e68768e5758c3293b2daf688
+# List of conda versions: https://repo.anaconda.com/miniconda/
+#
+# Miniconda 4.9.2 has been chosen as it has a aarch64 release and hasn't
+# introduced a bug described in:
+# https://stackoverflow.com/questions/68213186/illegal-instruction-error-when-verifying-anaconda-miniconda-install
+#
+# FIXME: Use micromamba or mambaforge instead, see
+#        https://github.com/mamba-org/mamba for more info
+#
+ARG CONDA_VERSION=py38_4.9.2
 
 # - Create user dask
+RUN useradd -m -U -u 1000 dask
+
 # - Install tini
 # - Install miniconda build dependencies
-# - Download miniconda and check the sha256 checksum
-# - Install miniconda
+RUN apt-get update \
+    && apt-get install -y tini wget bzip2 \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
I split this apart from a section below making so many different things in the same RUN layer that it became a bit hard to read.